fabric-events-player-v1/src/main/java/net/fabricmc/fabric/mixin/event/player/MixinPlayerManager.java

```diff
@@ -0,0 +1,37 @@
+package net.fabricmc.fabric.mixin.event.player;
+
+import net.minecraft.server.network.ServerPlayerInteractionManager;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.util.math.Vec3d;
+import org.objectweb.asm.Opcodes;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Inject;
+import org.spongepowered.asm.mixin.injection.Slice;
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfoReturnable;
+
+import net.minecraft.server.PlayerManager;
+import net.minecraft.server.network.ServerPlayerEntity;
+import net.minecraft.world.dimension.DimensionType;
+
+import net.fabricmc.fabric.api.event.player.v1.PlayerRespawnCallback;
+import org.spongepowered.asm.mixin.injection.callback.LocalCapture;
+
+@Mixin(PlayerManager.class)
+public abstract class MixinPlayerManager {
+	/**
+	 * The this injection has been chosen to allow repositioning of the player by dimension and location before respawn packets are sent to the client.
```

Comment:
```suggestion
	 * This injection point has been chosen to allow repositioning of the player by dimension and location before respawn packets are sent to the client.
```
missed that

Reply:
Well best to do the docs right now I guess